docs/api/scheduler.md

```diff
@@ -35,7 +35,7 @@ Under the `allocations` field in the response content for the app/node-related c
 
 ## Partitions
 
-Displays general information about the partition like name, state, capacity, used capacity, utilization, and node sorting policy.
```

Comment:
I think we can simplify this rather than listing every item, since we have examples. Maybe:

`Returns general information and statistics about a partition.`

Reply:
Done. Thanks.